### PR TITLE
add auto compile github action

### PR DESCRIPTION
**Pull request by YangKeao**
Signed-off-by: YangKeao <yangkeao@chunibyo.icu>

Add a comment "/build-image" to auto compile the image and comment the download URL.

See https://github.com/YangKeao/chaos-mesh/pull/2#issuecomment-1054001816 for the behavior.

TODO:
- [x] add a script to download, unzip and import the image in the comment.

### CODE REVIEW

**Comment by YangKeao** [comment]
@STRRL  PTAL

**Comment by YangKeao** [comment]
> /build-image

It will only take effect after merged into the master...

**Review by STRRL — APPROVED** [review]

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

File: hack/download-image.sh [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+set -e
+
+function usage() {
+    cat <<'EOF'
+This command downloads and imports the chaos-mesh image from the github artifacts
+
+Usage: hack/download-image.sh
+
+    -h show this message and exit
+    -r the github repository which running the
```
Comment (fewdan) [inline hack/download-image.sh:26]: "-r the github repository which running the " looks a little weird?

**Review by fewdan — APPROVED** [review]
rest LGTM, thanks

File: hack/download-image.sh (hunk 1 above)
Comment (YangKeao) [reply hack/download-image.sh]: Fixed.

**Pull request merged**